demo/index.html

```diff
@@ -100,6 +111,27 @@
                     <option value="3">Baz</option>
                 </select>
             </div>
+            <div class="demo demo-5">
+                <h2>Adding items</h2>
+                <p>Click the button to add a new item</p>
+                <select id="demo-5">
+                    <option value="1">Foo</option>
+                    <option value="2">Bar</option>
+                    <option value="3">Baz</option>
+                </select>
+                <button id="addButton">Click me!</button>
+                <input type="checkbox" id="select-immediately"></input><label for="select-immediately">Select the new item immediately</label>
```

Comment:
The alignment is a bit weird at least on chrome 
![image](https://cloud.githubusercontent.com/assets/167537/6270068/92b13d9c-b857-11e4-9cf2-97999a7d0017.png)